packages/common/src/Adaptor.js

```diff
@@ -932,3 +932,35 @@ export function assert(expression, errorMessage) {
     return state;
   };
 }
+
+
+/**
+ * Runs an operation and saves the return value to a pre-defined key in state
+ * @public
+ * @function
+ * @example
+ * as('lookup_table.option_set', get('fixture/?fixture_type=option_set_mapping'));
```

Comment:
This example is interesting. If the key has dot notation, should it be a path? This would be easy to implement with lodash.set. But I also don't know the use-case

Reply:
Let's go ahead and use lodash.set to support paths in `as`. I don't really know why, but this is more of a WHY NOT situation